cmd/medius/images/verify.go

```diff
@@ -113,6 +124,25 @@ func NewVerifyImagesCommand(options *common.Options) *cobra.Command {
 	return verifyCmd
 }
 
+func retrieveNodeArch(client kvirtcli.KubevirtClient) (string, error) {
+	nodes, err := client.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return "", err
+	}
+	if len(nodes.Items) == 0 {
+		return "", fmt.Errorf("no nodes found")
+	}
+	return nodes.Items[0].Status.NodeInfo.Architecture, nil
```

Comment:
How do we know the VM is going to be scheduled on this node / that arch? Perhaps we need to add a node selector for the architecture to ensure that.

Reply:
I think you make a good point here, but... Currently, we are enabling KubeVirt for various architectures, but not within multi-architecture clusters. I believe that this will come, though it will require a significant amount of work. Additionally, we have some control over our testing environment; for instance, our Prow worker is exclusively s390x.
Given these considerations, do you think it is reasonable to assume that we are only working with single-architecture clusters for the time being?

Reply:
I'm okay with that for the time being. Can you please open an issue so this TODO does not get lost?

Reply:
Issue created: https://github.com/kubevirt/containerdisks/issues/279